aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -15,35 +15,36 @@ module aptos_framework::coin {
     // Errors.
     //
 
-    /// When address of account which is used to initilize a coin `CoinType`
-    /// doesn't match the deployer of module containining `CoinType`.
-    const ECOIN_INFO_ADDRESS_MISMATCH: u64 = 0;
+    /// Address of account which is used to initilize a coin `CoinType` doesn't match the deployer of module
+    /// containining `CoinType`.
+    const ECOIN_INFO_ADDRESS_MISMATCH: u64 = 1;
 
-    /// When `CoinType` is already initilized as a coin.
-    const ECOIN_INFO_ALREADY_PUBLISHED: u64 = 1;
+    /// Specified `CoinType` is already initilized as a coin.
+    const ECOIN_INFO_ALREADY_PUBLISHED: u64 = 2;
 
-    /// When `CoinType` hasn't been initialized as a coin.
-    const ECOIN_INFO_NOT_PUBLISHED: u64 = 2;
+    /// Specified `CoinType` hasn't been initialized as a coin.
+    const ECOIN_INFO_NOT_PUBLISHED: u64 = 3;
 
-    /// When an account already has `CoinStore` registered for `CoinType`.
-    const ECOIN_STORE_ALREADY_PUBLISHED: u64 = 3;
+    /// Specified an account already has `CoinStore` registered for `CoinType`.
+    const ECOIN_STORE_ALREADY_PUBLISHED: u64 = 4;
 
-    /// When an account hasn't registered `CoinStore` for `CoinType`.
-    const ECOIN_STORE_NOT_PUBLISHED: u64 = 4;
+    /// Specified an account hasn't registered `CoinStore` for `CoinType`.
+    const ECOIN_STORE_NOT_PUBLISHED: u64 = 5;
 
-    /// When there's not enough funds to withdraw from an account or from `Coin` resource.
-    const EINSUFFICIENT_BALANCE: u64 = 5;
+    /// There's not enough funds to withdraw from an account or from `Coin` resource.
+    const EINSUFFICIENT_BALANCE: u64 = 6;
 
-    /// When destruction of `Coin` resource contains non-zero value attempted.
-    const EDESTRUCTION_OF_NONZERO_TOKEN: u64 = 6;
+    /// Cannot destroy a `Coin` resource that contains non-zero value.
+    const EDESTRUCTION_OF_NONZERO_TOKEN: u64 = 7;
 
     /// Total supply of the coin overflows. No additional coins can be minted.
-    const ETOTAL_SUPPLY_OVERFLOW: u64 = 7;
+    const ETOTAL_SUPPLY_OVERFLOW: u64 = 8;
 
-    const EINVALID_COIN_AMOUNT: u64 = 8;
+    /// Coin amount cannot be 0.
+    const EZERO_COIN_AMOUNT: u64 = 9;
 
     /// Coins cannot be deposited or withdrawn from this CoinStore.
```

Comment:
CoinStore is frozen.  Coins cannot be deposited or withdrawn.

Reply:
Updated